src/watchful/client.py

```diff
@@ -1080,6 +1080,19 @@ def export_preview(mode: str = "ftc") -> Optional[Dict]:
 
     return api("export_preview", mode=mode)
 
+def export_project() -> http.client.HTTPResponse:
+    """
+    This function returns a consolidated version (a single *.hints file) of the
+    currently open project.
+    """
+
+    conn = _get_conn()
+    conn.request(
+        "GET",
+        "/export_project")
+    resp = conn.getresponse()
+    assert 200 == int(resp.status), f"Request failed with status {resp.status}."
```

Comment:
`== resp.status`

Reply:
I'm not sure I follow what you're suggesting.  Are you saying an `int()` case is not necessary?

Reply:
Yes, I meant to simplify `assert 200 == int(resp.status)` to `assert 200 == resp.status`.

Reply:
ok. there were other places in the client as well where it was `int()` casting.  I've stripped those so they're all consistent.